### PR TITLE
RDKTV-12823: Option to set individual 'memorypressure' configs for Sa…

### PR DESCRIPTION
**Pull request by adrianM27**
…D, RA, HtmlApp and LightningApp

Reason for change: Option to set individual 'memorypressure'
 configs for SaD, RA, HtmlApp and LightningApp
Test Procedure: Refer ticket.
Risk:Low

Signed-off-by: Adrian Muzyka <adrian.muzyka@consult.red>

### CODE REVIEW

**Review by emutavchi — CHANGES_REQUESTED** [review]
Could you please set defaults in the CMakeLists.txt. default to ${PLUGIN_WEBKITBROWSER_MEMORYPRESSURE}

**Comment by adrianM27** [comment]
> Could you please set defaults in the CMakeLists.txt. default to ${PLUGIN_WEBKITBROWSER_MEMORYPRESSURE}

**Review by emutavchi — APPROVED** [review]
Looks good to me. Let's wait for @MFransen69 and @HaseenaSainul  review before merging.

**Review by HaseenaSainul — APPROVED** [review]

**Pull request merged**
